src/internal/observable/of.ts

```diff
@@ -79,5 +80,5 @@ export function of<A extends readonly unknown[]>(...values: A): Observable<Value
  */
 export function of<T>(...args: Array<T | SchedulerLike>): Observable<T> {
   const scheduler = popScheduler(args);
-  return from(args as T[], scheduler);
+  return scheduler ? scheduled(args as T[], scheduler) : fromArrayLike(args as T[]);
```

Comment:
`fromArrayLike` is not exported from `from`. Should this just be `from`?

Reply:
There can be nothing here but an array. `from` [contains a lot of code](https://github.com/ReactiveX/rxjs/blob/314b204acc9a8923830789c948727febb5b521e1/src/internal/observable/from.ts#L92-L109) that converts various types into `Observable` which will never be executed in the context of `of`.